app/org/sagebionetworks/bridge/models/schedules/SurveyReference.java

```diff
@@ -1,58 +1,58 @@
 package org.sagebionetworks.bridge.models.schedules;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import java.util.Objects;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 
 import org.joda.time.DateTime;
-import org.sagebionetworks.bridge.json.BridgeTypeName;
+import org.joda.time.format.ISODateTimeFormat;
+import org.sagebionetworks.bridge.config.BridgeConfigFactory;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
  * This is a "soft" reference to a survey that does not need to include a createdOn timestamp. 
  * It can be used to create JSON for published versions of surveys as well as hard references 
  * to a specific version.
  */
-@BridgeTypeName("GuidCreatedOnVersionHolder")
 public final class SurveyReference {
 
-    public static final String SURVEY_RESPONSE_PATH_FRAGMENT = "/surveyresponses/";
-    public static final String SURVEY_PATH_FRAGMENT = "/surveys/";    private static final String PUBLISHED_FRAGMENT = "published";
-    private static final String REGEXP = "http[s]?\\://.*/surveys/([^/]*)/revisions/([^/]*)";
```

Comment:
I'd remove this regex string to within the Pattern.compile() method below so that it guaranteed to use the compiled pattern.